format.md

```diff
@@ -34,16 +91,28 @@ Each transition line consists of:
 - The string "daylight" or "standard" to indicate whether the
   period after the transition is in daylight or standard time
 - A space
-- The abbreviated name of the zone at this time, e.g. "CST" or "CDT"
-
-Lines are separated by "\r\n" - the Windows default line
-separator - until such time as that becomes annoying. (It's
-convenient for me, Jon Skeet, as I mostly work on Windows...)
+- The abbreviated name of the zone at this time, e.g. "CST" or "CDT".
+  This is the abbreviation for the period from this transition to
+  the next transition.
+  
+As a complete example, the 2016c data should generate the following
+lines for the `America/La_Paz` zone. 
 
+    America/La_Paz
+    Initially:           -04:32:36 standard LMT
```

Comment:
Humbug - they're being squashed. They're in the source. Will be more explicit about them anyway, and see whether using `&nbsp;` fixes it.